src/libsyncengine/update_detection/file_system_observer/snapshot/snapshot.cpp

```diff
@@ -84,6 +85,12 @@ bool Snapshot::updateItem(const SnapshotItem &newItem) {
         startUpdate();
     }
 
+    if (ParametersCache::instance()->parameters().extendedLog()) {
```

Comment:
Out of scope: It would be nice to have s shorthand for this call, for instance: `ParametersCache::isLogExtended()`.

Reply:
Agreed. Task created in backlog: https://infomaniak.atlassian.net/browse/KDESKTOP-864